### ISSUE TITLE
normalize-by-median.py - ERROR: I/O operation on closed file

### ISSUE THREAD

**Issue by ljcohen** [dib-lab/khmer#1341]
Received the error below, looks similar to issue #1320. My original .fq file was combined from several runs to cut down on number of files to work with. Not sure if that makes a difference?

```
DONE with /home/ljcohen/osmotic_trim/interleave/F_rathbuni_NA_BW_2.trim.interleaved.fq.gz; kept 12095876 of 21838660 or 55.4%
output in /home/ljcohen/osmotic_trim/diginorm/F_rathbuni_NA_BW_2.trim.keep
** ERROR: I/O operation on closed file
** Failed on /home/ljcohen/osmotic_trim/F_rathbuni_NA_BW_2.trim.orphans.fq.gz: 
** Exiting!
```
This was the original command:
```
normalize-by-median.py -p -k 20 -C 20 -M 4e9 \
--savegraph /home/ljcohen/osmotic_trim/diginorm/F_rathbuni_NA_BW_2.trim.norm.C20k20.ct \
-o /home/ljcohen/osmotic_trim/diginorm/F_rathbuni_NA_BW_2.trim.keep \
-u /home/ljcohen/osmotic_trim/F_rathbuni_NA_BW_2.trim.orphans.fq.gz \
/home/ljcohen/osmotic_trim/interleave/F_rathbuni_NA_BW_2.trim.interleaved.fq.gz
```

**Comment by ctb** [comment]
Yes, this is the same as #1320; fixed in #1321.

**Comment by ljcohen** [comment]
Got it, thank you. Installed development version and works now.

```
|| This is the script normalize-by-median.py in khmer.
|| You are running khmer version 1.4+1085.ge6eed56
|| You are also using screed version 0.9
```
